<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevoVuelo</class>
 <widget class="QDialog" name="NuevoVuelo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="modificar">
       <property name="text">
        <string>Modificar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="guardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="line_destino"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Ingrese Ciudad Origen</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="line_distancia"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Precio</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Fecha</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ingrese Ciudad Destino</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="line_precio"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="line_origen"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Distancia</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Horario Llegada</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="line_llegada"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Horario Salida</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="line_salida"/>
   </item>
   <item row="4" column="1">
    <widget class="QCalendarWidget" name="line_calendario"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelar</sender>
   <signal>clicked()</signal>
   <receiver>NuevoVuelo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
